jekyll/_cci2/credits.md

```diff
@@ -35,15 +35,70 @@ To set up your desired plan, go to `Settings > Plan Overview` in the CircleCI we
 ## Free plan
 {: #free-plan }
 
-As with the CircleCI legacy Container plan, CircleCI also supports a free-tier with the usage-based plan. You can still make use of many key features offered by CircleCI:
-
-- Using Orbs
-- Workspaces
-- Dependency Caching
-- Windows/Linux builds
+As with the CircleCI legacy Container plan, CircleCI also supports a free-tier with the usage-based plan. By using the Free Plan, you can take advantage of a large number of premium features that will allow your team to be more productive, efficient and fast.
 
 The free usage-based plan offers a set amount of build credits across medium-type machines per week (which offers 2 CPUs, 4gb of memory). With this combination, a small number of credits are charged per minute and there is no limit on user seats. Refer to the [Pricing](https://circleci.com/pricing/) page for more information on credit amounts.
 
+The table below lists some of the features you will be able to use on the Free Plan.
+
+| Feature | Description |
+|-------|-------------|
+| UNLIMITED USERS | You may have an unlimited number of users. This enables you to have different teams and groups working together on projects |
+| EXPANSIVE LIST OF AVAILABLE RESOURCE CLASSES | Docker, Linux and Windows are available (macOS coming soon) are available for us |
```

Comment:
```suggestion
Docker, Linux and Windows (macOS coming soon) are available for us |
```
"are available" was repeated twice. Also, is it supposed to be "for use" (and not "for us")?

A bit nitpicky, but I prefer not to have the same word repeated too closely together, so I may remove "available" from the Feature column here i.e. "EXPANSIVE LIST OF RESOURCE CLASSES" - or, rephrase the description.